src/main/java/se/sundsvall/byggrintegrator/api/validation/ValidPersonalOrOrgNumber.java

```diff
@@ -0,0 +1,34 @@
+package se.sundsvall.byggrintegrator.api.validation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.hibernate.validator.constraints.CompositionType;
+import org.hibernate.validator.constraints.ConstraintComposition;
+
+import se.sundsvall.dept44.common.validators.annotation.ValidOrganizationNumber;
+import se.sundsvall.dept44.common.validators.annotation.ValidPersonalNumber;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+import jakarta.validation.ReportAsSingleViolation;
+
+@Documented
+@ConstraintComposition(CompositionType.OR)
+@Constraint(validatedBy = {})
+@ValidPersonalNumber
+@ValidOrganizationNumber
+@ReportAsSingleViolation
+@Target({ ElementType.FIELD, ElementType.PARAMETER })
+@Retention(RetentionPolicy.RUNTIME)
+public @interface ValidPersonalOrOrgNumber {
```

Comment:
This works, but you could also re-use existing `@ValidPersonalNumber` and `@ValidOrganizationNumber` with `org.hibernate.validator.constraints.ConstraintComposition`. See org.hibernate.validator.constraints.ConstraintComposition